blocks/editor-settings/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext } from '@wordpress/element';
+
+/**
+ * The default editor settings
+ * You can override any default settings when calling initializeEditor
+ *
+ *  alignWide   boolean   Enable/Disable Wide/Full Alignments
+ *
+ * @var {Object} DEFAULT_SETTINGS
```

Comment:
I don't know that `@var` is correct here, and I'm not sure why its value is repeated `DEFAULT_SETTINGS`.